### PR TITLE
fix: move exp tracking to mounted hook and remove eligibility gating …

### PR DESCRIPTION
**Pull request by mcstover**
…around analytics on lend/filter

### CODE REVIEW

File: src/pages/Lend/Filter/LendFilterPage.vue [hunk 1]
```diff
@@ -208,20 +208,6 @@ export default {
 
 		// Get Lend Filter Exp version
 		this.getLendFilterExpVersion();
-
-		if (isFLSSEligible(this.$route)) {
```
Comment (mcstover) [inline src/pages/Lend/Filter/LendFilterPage.vue:212]: This extra check around the tracking call is the primary issue as the Algolia components immediately push in the `popularity` sort param causing the url read to be in-eligible by the time this is caused.

File: src/pages/Lend/LoanSearchPage.vue [hunk 2]
```diff
@@ -70,7 +70,7 @@ export default {
 				});
 		}
 	},
-	created() {
+	mounted() {
```
Comment (dyersituations) [inline src/pages/Lend/LoanSearchPage.vue:73]: Why was this change needed?

File: src/pages/Lend/Filter/LendFilterPage.vue [hunk 3]
```diff
@@ -316,6 +302,20 @@ export default {
 
 		// update global lend filter experiment setting
 		this.updateLendFilterExp();
+
+		// Check cached for lend/filter-alpha experiment state and track if present
```
Comment (dyersituations) [inline src/pages/Lend/Filter/LendFilterPage.vue:306]: No need to check `isFLSSEligible`?

**Review by dyersituations — APPROVED** [review]

File: src/pages/Lend/LoanSearchPage.vue (hunk 2 above)
Comment (mcstover) [reply src/pages/Lend/LoanSearchPage.vue:73]: Analytics calls can only execute in the client, so it's a more explicit location to attempt the call. More preference than necessity.
